packages/hooks/src/stories/UseTokenBalance.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Provider, useWallet } from '..';
+import { useTokenBalance } from '../hooks/useTokenBalance';
+
+const Default = () => {
+  const { connectWallet, connected } = useWallet();
+
+  if (connected) {
+    return (
+      <>
+        <BalanceOfDai />
+        <UnknownOwner />
+        <UnknownContract />
+      </>
+    );
+  }
+
+  return <button onClick={connectWallet}>Connect Wallet</button>;
+};
+
+storiesOf('useTokenBalance', module).add('Default', () => (
+  <Provider network='mainnet'>
```

Comment:
should we use a non mainnet for stories? I think we used rinkeby for other stories